src/Smidge.Core/Options/BundleOptions.cs

```diff
@@ -80,6 +81,11 @@ public Type GetCacheBusterType()
         /// </summary>
         public bool CompressResult { get; set; }
 
+        /// <summary>
+        /// The compression level of the bundle
+        /// </summary>
+        public CompressionLevel CompressionLevel {get; set; }
```

Comment:
Is this addressing another issue/feature?

Reply:
Yes, I have added this as Smidge was hardcoding this to Optimal. This setting is usually preferred, but I'd like this configurable for bundles so I can use Fastest for local testing.

Reply:
Okidoke, IIRC I think there is a ticket on the tracker for this which may be able to be resolved then.

Reply:
#135 should be that, I'll add that to description so it can be closed.